packages/@aws-cdk/aws-codepipeline/README.md

```diff
@@ -80,16 +96,91 @@ or you can use the `IStage.addAction()` method to mutate an existing Stage:
 sourceStage.addAction(someAction);
 ```
 
+### Cross-account CodePipelines
+
+Cross-account Pipeline actions require that the Pipeline has *not* been
+created with `crossAccountKeys: false`.
+
+To perform actions in a different account than your Pipeline is in, most
+actions accept a resource that is in a different account (either created
+or imported):
+
+```typescript
+stage.addAction(new codepipeline_actions.S3DeployAction({
+  bucket: s3.Bucket.fromBucketAttributes(this, 'Bucket', {
+    account: '123456789012',
+    // ...
+  }),
+  // ...
+}));
+```
+
+Some actions accept an explicit `account` parameter:
+
+```typescript
+stage.addAction(new codepipeline_actions.CloudFormationCreateUpdateStackAction({
+  account: '123456789012',
+  // ...
+}));
+```
+
+CodePipeline requires that an IAM Role exists in the target account with a
+well-known name. The `Pipeline` construct automatically defines a **support
+stack** for you, named `<PipelineStackName>-support-<account>`, that will
+provision a role that the pipeline will assume in the given account before
+executing this action. This support stack will automatically be deployed
+before the stack containing the pipeline.
+
+You can also explicitly pass a `role` when creating the action. In that case,
+the `account` property is ignored, and the action will operate in the same
+account the role belongs to:
+
+```ts
+stage.addAction(new codepipeline_actions.CloudFormationCreateUpdateStackAction({
+  // ...
+  role: iam.Role.fromRoleArn(this, 'ActionRole', '...'),
+}));
+```
+
 ### Cross-region CodePipelines
 
-You can also use the cross-region feature to deploy resources
-into a different region than your Pipeline is in.
+To perform actions in a different region than your Pipeline is in, most
+actions accept a resource that is in a different region (either created
+or imported):
+
+```typescript
+stage.addAction(new codepipeline_actions.S3DeployAction({
+  bucket: s3.Bucket.fromBucketAttributes(this, 'Bucket', {
+    region: 'us-west-1',
+    // ...
+  }),
```

Comment:
Repeating content.

Reply:
? Isn't the difference cross-account and cross-region?

Reply:
Oops, missed that. Feels like this can be merged into the same section. And change the phrasing to 'account or region or both'.

Up to you.